externs/shaka/player.js

```diff
@@ -615,12 +615,16 @@ shaka.extern.DashManifestConfiguration;
 
 /**
  * @typedef {{
- *   ignoreTextStreamFailures: boolean
+ *   ignoreTextStreamFailures: boolean,
+ *   forceFullSegmentRequest: boolean
  * }}
  *
  * @property {boolean} ignoreTextStreamFailures
  *   If <code>true</code>, ignore any errors in a text stream and filter out
  *   those streams.
+ * @property {boolean} forceFullSegmentRequest
```

Comment:
Also, it would probably be better to name this property "useFullSegmentsForStartTime".

We plan on working on an LL-HLS feature in the future, which will involve fetching partial segments. It'd be best to rename this, so that once we have LL-HLS it's clear that this option isn't related.

Reply:
renamed